packages/orbit-components/src/CountryFlag/__tests__/index.test.tsx

```diff
@@ -42,4 +43,36 @@ describe("CountryFlag", () => {
     const flag = screen.getByRole("img");
     expect(flag).toHaveAttribute("src", expect.stringContaining("us"));
   });
+  it("should have the correct styles", () => {
```

Comment:
it looks a bit dirty too me :) 

better will be avoid imperative style with let and cleanup. I suggest to split on two different tests, something like:
- it should have expected DOM output / styles 
- it should have correct size